### DISCUSSION TITLE
Compressor/expander

### DISCUSSION THREAD

**Discussion by mrx23dot** [pschatzmann/arduino-audio-tools#1027] [Q&A] [answered]
Any plans to include Compressor/expander into this lib?
With standard parameters like attack/release, see https://www.youtube.com/watch?v=SUd4U87EcDo

I was planning to do a night movie mode (dynamic range compression) for i2s signal.

**Comment by pschatzmann** [comment, marked as answer]
Not really, but I have added a DRAFT [Compressor](https://pschatzmann.github.io/arduino-audio-tools/classaudio__tools_1_1_compressor.html) based on https://github.com/YetAnotherElectronicsChannel/STM32_DSP_COMPRESSOR

Feel free to extend/correct it or provide your own implementation.

**Reply by mrx23dot** [reply]
Looks interesting, thank you!
